### ISSUE TITLE
Can there be multiple request interceptors  defined with axiosInstance?

### ISSUE THREAD

**Issue by sauravmishra1710** [axios/axios#4861]
#### Describe the issue
This is more of an information and knowledge question.
Can we define 2 different request interceptors for a single axiosInstance? Currently I have defined 2 interceptors on the axiosInstance but only the 1st one seems to be functional and intercepting the requests. The 2nd one never comes into picture. So I wanted to understand if there can be such a scenario to define multiple interceptors?

#### Example Code
Code snippet to illustrate your question

```js
// Example code here
```

#### Environment
 - Axios Version [e.g. 0.18.0]
 - Adapter XHR
 - Browser Chrome
 - Browser Version 103.0.5060.114 
 - Node.js Version 10.16
 - OS: [e.g. iOS 12.1.0, OSX 10.13.4]
 - Additional Library Versions [e.g. React 16.7, React Native 0.58.0]

#### Additional context/Screenshots
Add any other context about the problem here. If applicable, add screenshots to help explain.

**Comment by khobiziilyes** [comment]
Please refer to [Multiple interceptors](https://github.com/axios/axios#multiple-interceptors)

**Comment by sauravmishra1710** [comment]
@khobiziilyes I went through the multiple interceptors and implemented 2 interceptors (say for eg. intercept1 and intercept2). The documentation readme says - _"then they are executed in the order they were added"_. So ideally they should be executed as intercept1 and then intercept2 executes. 
However, I am seeing the reverse. For me intercept2 executes first and then intercept1. Is this expected? or something is not right here?
